### ISSUE TITLE
CPU usage is getting extreamly high by using snapcheck module

### ISSUE THREAD

**Issue by taijiji** [Juniper/jsnapy#204]
Hi,

I found the fatal issue for using Python module.
When I try to run Python script using snapcheck module, CPU usage was getting extreamly high and It couldn't finish the script.

I show my environment and debug code.

PC : MacBook Air (1.7GHz,Intel Core i7, Mid 2013 model)
router : firefly-perimeter, JUNOS 12.1X47-D20.7
Program : Python 2.7.10, JSNAPy 1.0.0

```:run_jsnapy_for_debug.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

from jnpr.junos import Device
from jnpr.junos.utils.config import Config
from jnpr.jsnapy import SnapAdmin

from pprint import pprint, pformat


jsnapy_config =\
'''
hosts:
  - device: 192.168.34.16 
    username : user1
    passwd: password1
    port: 22
tests:
- ./tests/test_hostname.yml
'''

jsnapy = SnapAdmin()

for i in range(0,10):
    print i
    snapcheck_dict = jsnapy.snapcheck(
                  data = jsnapy_config,
                  file_name = "snap01")
```

```:./tests/test_hostname.yml
test_hostname:
  - command: show version
  - item:
      xpath: '/software-information'
      tests:
        - is-equal: host-name, firefly1
          info: "Test : OK, host-name is <{{pre['host-name']}}>"
          err:  "Test : NG, host-name is <{{pre['host-name']}}>"
```

How to run this code

```
python run_jsnapy_for_debug.py
```

output

```
% python run_jsnapy_for_debug.py                                                                                                                                                             

0
1
2

(Can't proceed. PC was frozen. CPU usage was over 90%.)
```

**Comment by eeishaan** [comment]
Hi @taijiji 

I suppose you are using jsnapy v1.0 . We have fixed this issue ( #143 and #157 ) in pull request #166 as part of v1.1. v1.1 is not released yet. But you can still update your jsnapy to the latest code by running the following `sudo pip install -U git+https://github.com/Juniper/jsnapy.git` and this issue will go away

**Comment by taijiji** [comment]
Hi @eeishaan 

I tried to use jsnapy (1.0.1) and I have confirmed that this issue has solved.
Appreciate your support!
